runelite-client/src/main/java/net/runelite/client/plugins/microbot/Script.java

```diff
@@ -122,16 +124,18 @@ public boolean run() {
                 Rs2Keyboard.keyPress(KeyEvent.VK_SPACE);
             }
             Widget clickHereToPlayButton = Rs2Widget.getWidget(24772680); //on login screen
-            if (clickHereToPlayButton != null && !Microbot.getClientThread().runOnClientThread(clickHereToPlayButton::isHidden)) {
+            if (clickHereToPlayButton != null && !Microbot.getClientThread().runOnClientThread(clickHereToPlayButton::isHidden) && !clickedPlayButton) {
                 Rs2Widget.clickWidget(clickHereToPlayButton.getId());
+                clickedPlayButton = true;
```

Comment:
Updated the code to use sleepUntil instead. I also use a synchronized block to prevent multiple plugins from clicking the play button at the same time